internal/openmetrics-exporter/buckets_s3_perf_collector.go

```diff
@@ -7,16 +7,15 @@ import (
 )
 
 type BucketsS3PerfCollector struct {
-	LatencyDesc     *prometheus.Desc
-	ThroughputDesc  *prometheus.Desc
-	BandwidthDesc   *prometheus.Desc
-	AverageSizeDesc *prometheus.Desc
-	Client          *client.FBClient
-	Buckets         *client.BucketsList
+	LatencyDesc    *prometheus.Desc
+	ThroughputDesc *prometheus.Desc
+	Client         *client.FBClient
+	Buckets        *client.BucketsList
```

Comment:
Good shout on finding these unused fields. This may need to be put into a separate PR.